### DISCUSSION TITLE
https://plugin-components.vuejs.press/zh/guide/addthis.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#616] [Announcements]
# https://plugin-components.vuejs.press/zh/guide/addthis.html

这是一个全局组件，提供了分享功能和访客统计功能，基于 AddThis (https://www.addthis.com/)。 注册 前往 Add this 注册页面 (https://www.addthis.com/register?next=/dashboard) 注册 如果你已注册，前往 Add this 登录页面 (https://www.add...

https://plugin-components.vuejs.press/zh/guide/addthis.html

**Comment by ifengshai** [comment]
自 2023 年 5 月 31 日起 AddThis 服务已被终止。

**Reply by Mister-Hope** [reply]
可以开个issue
